src/clims/devhelpers/generate.py

```diff
@@ -0,0 +1,93 @@
+from __future__ import absolute_import
+
+import os
+
+from clims.devhelpers import resources
+import re
+from jinja2 import Template
+import stringcase
+
+# TODO: This should be a separate module (e.g. clims_devtools)
+
+
+class CodeGenerator(object):
+    """
+    Generates starting point code for the CLIMS project. Given a resource name, generates:
+        * API endpoint for listing the resource
+        * API endpoint for details
```

Comment:
I guess these are not true yet? All I see so far is code to generate the redux actions.

Reply:
Yeah, these are just empty promises at the moment :P I'll throw in TODOS.